kpet/run.py

```diff
@@ -106,34 +106,21 @@ def __get_recipesets(database, target):
         return recipesets_of_hosts
 
     @staticmethod
-    def __get_hosts(database, target):
+    def __distribute_into_hosts(database, pool_suites):
         """
-        Get a list of hosts to run.
+        Distribute suites and their cases to hosts
 
         Args:
-            database:   The database to get test data from.
-            target:     The target (a data.Target) to match/run tests against.
+            database:       The database to get host_types from.
+            pool_suites:    The pool of suites to distribute.
```

Comment:
Might be better to specify the exact data type (a list of `(suite, list of its cases)` sets I think?)